libopenage/engine.h

```diff
@@ -149,7 +149,7 @@ class Engine : public ResizeHandler, public options::OptionNode {
 	/**
 	 * engine move operator.
 	 */
-	Engine &operator=(Engine &&other);
+	Engine &operator=(Engine &&other) noexcept;
```

Comment:
when I look at it, i can't find an implementation of this method, so you can remove the `noexcept` and `= delete` it as well.